lmcache/config.py

```diff
@@ -68,6 +71,23 @@ def from_file(file_path: str) -> 'LMCacheEngineConfig':
         remote_url = config.get("remote_url", None)
         remote_serde = config.get("remote_serde", "torch")
         pipelined_backend = config.get("pipelined_backend", False)
+        
+        match local_device:
+            case "cpu" | "cuda":
+                local_device = backend
+                remote_url = None
```

Comment:
Why set `remote_url` to `None` here? Does that mean we cannot use local and remote backend together?

In `from_legacy()` we set `remote_url` to None, because that interface is mainly used in old unit tests and it only takes one `backend` argument as input. I guess we don't need to do the same thing in `from_file()`